docs/userguide.md

```diff
@@ -276,19 +277,24 @@ you can patch role `kubefed-role` in the kubefed namespace instead.
 
 ### Disabling federation of an API type
 
-You can disable propagation of an API type using the
-`kubefedctl` command.
+You can disable propagation of an API type by editing its `FederatedTypeConfig`
+resource:
 
 ```bash
-kubefedctl disable <FederatedTypeConfig Name>
+kubectl patch --namespace <KUBEFED_SYSTEM_NAMESPACE> federatedtypeconfigs <NAME> \
+    --type=merge -p '{"spec": {"propagation": "Disabled"}}'
 ```
 
-This command sets the `propagationEnabled` field in the `FederatedTypeConfig`
-associated with this target API type to `false`, which will prompt the sync controller for the target API type to be stopped.
+This patch command sets the `propagation` field in the `FederatedTypeConfig`
+associated with this target API type to `Disabled`, which will prompt the sync
+controller for the target API type to be stopped.
 
-If you want to permanently disable federation of the target API type, pass the
-`--delete-crd` flag to remove the `FederatedTypeConfig` and federated type CRD created by
-`enable`.
+### Deleting federation of an API type
+
+If you want to permanently delete federation of the target API type by
+removing the `FederatedTypeConfig`, and optionally delete the federated type
+CRD created by `enable` using the `--delete-crd` flag, run the following
+command:
 
 ```bash
 kubefedctl disable <FederatedTypeConfig Name> --delete-crd
```

Comment:
Maybe if --delete-crd is rendered as optional [--delete-crd] a user will be less likely to copy-n-paste this data destroyer?

The warning below should state that '...supplying the --delete-crd flag will remove all custom resources...'

Reply:
I can make this change to make it more clear but as of #769 we do not allow deleting the CRD if custom resources exist.